tests/common.go

```diff
@@ -4967,6 +4967,29 @@ func StartTorpedoTest(testName, testDescription string, tags map[string]string,
 	tags["pureVolume"] = fmt.Sprintf("%t", Inst().PureVolumes)
 	tags["pureSANType"] = Inst().PureSANType
 	dash.TestCaseBegin(testName, testDescription, strconv.Itoa(testRepoID), tags)
+	EnableFeatures()
+
+}
+
+// EnableFeatures to enable px features
+func EnableFeatures(errChan ...*chan error) {
```

Comment:
function method says enable featues but it is enabling only autofstrim. can we genralise the method to features to enable as input (need to check if it is already there) or rename the function name

Reply:
renamed method EnableFaeture to "SetClusterOptions" , please note the  method is written such that this can be used for  setting other cluster options if required in future.
Also Added getClusterOpt()  to check cluster option is available and not set.